### PR TITLE
Use Service::getElementType() and trigger deprecated warning

### PR DESCRIPTION
**Pull request by blankse**

### CODE REVIEW

**Comment by brusch** [comment]
Very nice, thanks a lot!

File: models/Element/Service.php [hunk 1]
```diff
@@ -591,6 +591,12 @@ public static function getElementHash(ElementInterface $element): ?string
      */
     public static function getType($element)
     {
+        trigger_deprecation(
+            'pimcore/pimcore',
+            '10.0',
```
Comment (jdreesen) [inline models/Element/Service.php:596]: Shouldn't this be `10.2`? 
Docs say: `@param string $version The version of the package that introduced the deprecation`

Comment (blankse) [reply models/Element/Service.php:596]: Is deprecated since 10.0

Comment (brusch) [reply models/Element/Service.php:596]: yep, just `trigger_deprecation()` was missing 😉

**Pull request merged**